src/proc/Vigad.ts

```diff
@@ -76,4 +84,20 @@ export class Vigad {
     public getAllCaptureAreas(): CaptureArea[] {
         return this.captureAreas;
     }
+
+    public startTesseract(): void {
+        if (!this.intervalRunning) {
+            this.tesseractInterval = setInterval(() => {
+                this.tesseractHandler.run(this.streamHandler.getCurrentSelectedSource())
+            }, 500);
+            this.intervalRunning = true;
+            console.log("started tesseract");
+        }
+    }
+
+    public stopTesseract(): void {
+        clearInterval(this.tesseractInterval);
+        this.intervalRunning = false;
+        console.log("stopped tesseract");
+    }
```

Comment:
Before merging, remove console.log

Reply:
I actually kept them on purpose as I think it's something that can be quite useful for debugging. Of course a real logger would be better but this doesn't hurt, does it?